src/proto/Waves.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package TW.Waves.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+    int64 amount = 1;
+
+    string amount_asset = 2;
+
+    int64 fee = 3;
+
+    string fee_asset = 4;
+
+    string to = 5;
+
+    bytes attachment = 6;
```

Comment:
what is `attachment`? can you add some comments?

Reply:
some extra info of tx.
it will be shown in the wallet ui as utf-8 string, but can be also any bytes for some automatization stuff

Reply:
Added the comment